mscore/importmxmlpass2.h

```diff
@@ -147,10 +161,11 @@ class MxmlLogger;
 
 using SlurStack = std::array<SlurDesc, MAX_NUMBER_LEVEL>;
 using TrillStack = std::array<Trill*, MAX_NUMBER_LEVEL>;
-using BracketsStack = std::array<SLine*, MAX_BRACKETS>;
-using DashesStack = std::array<SLine*, MAX_DASHES>;
-using OttavasStack = std::array<SLine*, MAX_NUMBER_LEVEL>;
-using HairpinsStack = std::array<SLine*, MAX_NUMBER_LEVEL>;
+using BracketsStack = std::array<MusicXmlExtendedSpannerDesc, MAX_NUMBER_LEVEL>;
```

Comment:
With this change `BracketsStack` becomes decreased in its length: before this change it contained `MAX_BRACKETS` (which equals to 9) items while now it contains `MAX_NUMBER_LEVEL` (which is 6). Is this change intentional?

Reply:
Yes, as it makes the importer comply with the spec, which states that the number must be between 1 and 6. See: http://usermanuals.musicxml.com/MusicXML/MusicXML.htm#ST-MusicXML-number-level.htm. Larger numbers in MusicXML files should lead to validation errors on import, followed by ignoring the elements whose numbers are too high. If MuseScore allows it (don't know, would have to check), the limits for all stacks could be set higher. In that case the validation error would remain, but the import would be as expected. In retrospect, this might be better, but I'd suggest to make this explicit ib a separate issue.

Reply:
Created issue https://musescore.org/en/node/297357.